lms/djangoapps/dashboard/sysadmin.py

```diff
@@ -480,6 +485,173 @@ def post(self, request):
             branch = request.POST.get('repo_branch', '').strip().replace(' ', '').replace(';', '')
             self.msg += self.get_course_from_git(gitloc, branch)
 
+        elif action == 'download_course_users':
+            course_id = request.POST.get('course_id', '').strip()
+            course_key = SlashSeparatedCourseKey.from_deprecated_string(course_id)
+            course_found = False
+            if course_key in courses:
+                course_found = True
+                course = courses[course_key]
+            else:
+                try:
+                    course = get_course_by_id(course_key)
+                    course_found = True
+                except Exception, err:   # pylint: disable=broad-except
+                    self.msg += _(
+                        'Error - cannot get course with ID {0}<br/><pre>{1}</pre>'
+                    ).format(
+                        course_key,
+                        escape(str(err))
+                    )
+            if course_found:
```

Comment:
I can't even see where this if statement ends. Usually if want to do something only if some condition is true, it is more readable to exit if it is false and then put the logic for if it is true.